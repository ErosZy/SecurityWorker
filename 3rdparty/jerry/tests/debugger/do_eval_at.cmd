eval_at 0
eval_at 0 b
n
eval_at 0 b
b do_eval_at.js:20
n
scopes
eval_at 0 b
eval_at 1 b
eval_at 0 b=20
eval_at 1 b=100
n
eval_at 0 a
scopes
eval_at 0 b
eval_at -1 b
eval_at 65536 b
eval_at b
eval_at 200
c
