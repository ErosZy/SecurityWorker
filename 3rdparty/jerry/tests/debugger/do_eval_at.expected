Connecting to: localhost:5001
Stopped at tests/debugger/do_eval_at.js:15
(jerry-debugger) eval_at 0
undefined
(jerry-debugger) eval_at 0 b
undefined
(jerry-debugger) n
Stopped at tests/debugger/do_eval_at.js:23
(jerry-debugger) eval_at 0 b
2
(jerry-debugger) b do_eval_at.js:20
Breakpoint 1 at tests/debugger/do_eval_at.js:20 (in f() at line:17, col:1)
(jerry-debugger) n
Stopped at breakpoint:1 tests/debugger/do_eval_at.js:20 (in f() at line:17, col:1)
(jerry-debugger) scopes
level | type   
0     | local  
1     | global 
(jerry-debugger) eval_at 0 b
6
(jerry-debugger) eval_at 1 b
2
(jerry-debugger) eval_at 0 b=20
20
(jerry-debugger) eval_at 1 b=100
100
(jerry-debugger) n
Stopped at tests/debugger/do_eval_at.js:25
(jerry-debugger) eval_at 0 a
23
(jerry-debugger) scopes
level | type   
0     | global 
(jerry-debugger) eval_at 0 b
100
(jerry-debugger) eval_at -1 b
Error: Invalid scope chain index: -1 (must be between 0 and 65535)
(jerry-debugger) eval_at 65536 b
Error: Invalid scope chain index: 65536 (must be between 0 and 65535)
(jerry-debugger) eval_at b
Error: invalid literal for int() with base 10: 'b'
(jerry-debugger) eval_at 200
Uncaught exception: Error
(jerry-debugger) c
